instagram_own_version/permissions.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import permissions
+
+
+class IsOwner(permissions.BasePermission):
+    def has_permission(self, request, view):
+        return request.user.is_authenticated
+
+    def has_object_permission(self, request, view, obj):
```

Comment:
We do need to check if the request method is safe (ex. GET request) and allow safe requests
```
from rest_framework.permissions import SAFE_METHODS

if request.method in SAFE_METHODS:
...
```